token_auth/views/password_reset.py

```diff
@@ -0,0 +1,30 @@
+from django.shortcuts import render, redirect, reverse
+from rest_framework import generics, status, views, permissions
+from token_auth.serializers.password_reset import ResetPasswordEmailRequestSerializer
+from rest_framework.response import Response
+from django.contrib.sites.shortcuts import get_current_site
+from django.contrib.auth.models import User
+from django.contrib.auth.tokens import PasswordResetTokenGenerator
+from django.utils.encoding import smart_str, force_str, smart_bytes, DjangoUnicodeDecodeError
+from django.utils.http import urlsafe_base64_decode, urlsafe_base64_encode
+from token_auth.utils import Util
+
+
+class RequestPasswordResetEmailView(generics.GenericAPIView):
```

Comment:
Can you please add a doc string (in code documentation) explaining what this test is testing. E.g.: Given user that is registered, when _______, then ______. Just an example, you don't need to follow this format.

Reply:
Done